### PR TITLE
Don't sync _static dir for search builder

### LINKED ISSUE #3799: Stop syncing `_static` directory for JSON
Currently we are syncing the `_static` directory on our JSON builder. We should stop moving this across to the webs, because we never use it, and it's taking up _a ton_ of disk space. I've deleted it manually, but we should also make sure we stop syncing it in the future.

### PR DESCRIPTION
**Pull request by stsewd**
Closes #3799 

Was a little hard to figure out testing here, I think we don't have tests for the other builders :thinking:

### CODE REVIEW

**Comment by stsewd** [comment]
I did a manual test too :)

File: readthedocs/doc_builder/base.py [hunk 1]
```diff
@@ -63,7 +65,12 @@ def move(self, **__):
             if os.path.exists(self.target):
                 shutil.rmtree(self.target)
             log.info('Copying %s on the local filesystem', self.type)
-            shutil.copytree(self.old_artifact_path, self.target)
+            log.info('Ignoring patterns %s', self.ignore_patterns)
+            shutil.copytree(
+                self.old_artifact_path,
+                self.target,
+                ignore=shutil.ignore_patterns(*self.ignore_patterns)
```
Comment (stsewd) [inline readthedocs/doc_builder/base.py]: This is from https://docs.python.org/2/library/shutil.html#shutil.ignore_patterns

**Comment by ericholscher** [comment]
This is an elegant solution to this problem. :heart:

**Comment by ericholscher** [comment]
This is an elegant solution to this problem. :heart:

**Pull request merged**